modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/taur_types.dm

```diff
@@ -128,3 +128,33 @@
 	color_src = USE_ONE_COLOR
 	extra = TRUE
 	extra_color_src = MUTCOLORS2
+
+/datum/sprite_accessory/taur/chinesedragon
+	name = "Chinese Dragon"
+	icon_state = "noodle"
+	taur_mode = STYLE_TAUR_PAW
+
+/datum/sprite_accessory/taur/tentacle2
+	name = "Tentacle Alt"
+	icon_state = "tentaclealt"
+	taur_mode = STYLE_TAUR_SNAKE
```

Comment:
Something tells me they're not meant to be treated like a naga tail.

Reply:
The other tentacle one is.